### PR TITLE
implement rippleRadius for TouchableNativeFeedback

### PR DESCRIPTION
**Pull request by vonovak**
hi! this is a combination of 

https://github.com/facebook/react-native/pull/28600 and https://github.com/facebook/react-native/pull/28009

please let me know if you need any extra information, thank you!

### CODE REVIEW

File: android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java [hunk 1]
```diff
@@ -55,21 +58,31 @@ public void setRippleColor(Integer color) {
       mNeedBackgroundUpdate = true;
     }
 
+    public void setRippleRadius(Integer radius) {
+			mRippleRadius = radius;
+      mNeedBackgroundUpdate = true;
+    }
+
     public void setBorderRadius(float borderRadius) {
-      mBorderRadius = borderRadius * (float)getResources().getDisplayMetrics().density;
```
Comment (vonovak) [inline android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java:59]: the float cast was not needed

File: android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java [hunk 2]
```diff
@@ -55,21 +58,31 @@ public void setRippleColor(Integer color) {
...
+      mBorderRadius = borderRadius * getResources().getDisplayMetrics().density;
       mNeedBackgroundUpdate = true;
     }
 
     private Drawable applyRippleEffectWhenNeeded(Drawable selectable) {
       if (mRippleColor != null
-              && selectable != null
```
Comment (vonovak) [inline android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java:65]: `selectable != null` is covered by `selectable instanceof RippleDrawable`

**Comment by vonovak** [comment]
@osdnk @kmagiera sorry to ping you, would you be able to review (and hopefully release) this soon so that RNGH is on-par with RN 63? Thanks!

File: android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java [hunk 3]
```diff
@@ -225,6 +249,11 @@ public void setRippleColor(ButtonViewGroup view, Integer rippleColor) {
     view.setRippleColor(rippleColor);
   }
 
+  @ReactProp(name = "rippleRadius")
+  public void setRippleRadius(ButtonViewGroup view, Integer rippleColor) {
```
[the remark this answers is not in the document]
Comment (vonovak) [reply android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.java]: @jakub-gonet thanks, done :)

**Comment by vonovak** [comment]
@jakub-gonet any plans for a new release? Thank you!

**Comment by jakub-gonet** [comment]
After gathering few more PRs we'll release the next version. Thanks for contributing 🎉

**Pull request merged**